src/services/utilServices/SendDNSRecordEmailClient.ts

```diff
@@ -1,5 +1,11 @@
-import { groupBy } from "lodash"
+import { groupBy, has } from "lodash"
```

Comment:
nit: could we remove the `has` import - seems to be unused

Reply:
oops, removed in latest commit